### PR TITLE
[PRM-158] - Play 2.8 upgrade

### PR DESCRIPTION
**Pull request by LewisMitchell**

### CODE REVIEW

File: project/AppDependencies.scala [hunk 1]
```diff
@@ -6,23 +6,22 @@ import sbt._
...
+    "uk.gov.hmrc"             %% "bootstrap-frontend-play-28" % "5.3.0",
+    "uk.gov.hmrc"             %% "play-frontend-hmrc"         % "0.64.0-play-28",
+    "uk.gov.hmrc"             %% "play-frontend-govuk"        % "0.71.0-play-28",
+    "uk.gov.hmrc"             %% "play-language"              % "5.0.0-play-28"
   )
 
   val test = Seq(
-    "uk.gov.hmrc"             %% "bootstrap-test-play-27"         % "4.2.0"            % "test, it",
+    "uk.gov.hmrc"             %% "bootstrap-test-play-28"         % "5.3.0"            % "test, it",
     "org.scalatest"           %% "scalatest"                      % "3.0.8"            % "test, it",
```
Comment (AndrewRButcher) [inline project/AppDependencies.scala:17]: Version can be increased
```suggestion
    "org.scalatest"           %% "scalatest"                      % "3.0.9"            % "test, it",
```

File: project/plugins.sbt [hunk 2]
```diff
@@ -1,10 +1,10 @@
 resolvers += "HMRC-open-artefacts-maven" at "https://open.artefacts.tax.service.gov.uk/maven2"
 resolvers += Resolver.url("HMRC-open-artefacts-ivy", url("https://open.artefacts.tax.service.gov.uk/ivy2"))(Resolver.ivyStylePatterns)
 
-addSbtPlugin("uk.gov.hmrc"       % "sbt-auto-build"         % "2.14.0")
+addSbtPlugin("uk.gov.hmrc"       % "sbt-auto-build"         % "3.0.0")
 addSbtPlugin("uk.gov.hmrc"       % "sbt-git-versioning"     % "2.2.0")
```
Comment (AndrewRButcher) [inline project/plugins.sbt:5]: Can be removed with updated auto-build
```suggestion
```

**Review by AndrewRButcher — CHANGES_REQUESTED** [review]
Approved but a couple of dependency updates can be done

File: project/AppDependencies.scala (hunk 1 above)
Comment (LewisMitchell) [reply project/AppDependencies.scala]: Sorted.

File: project/plugins.sbt (hunk 2 above)
Comment (LewisMitchell) [reply project/plugins.sbt]: Sorted.

**Review by AndrewRButcher — APPROVED** [review]

**Pull request merged**